app/uk/gov/hmrc/tariffclassificationfrontend/models/CaseReport.scala

```diff
@@ -24,12 +24,14 @@ case class CaseReport
 (
   filter: CaseReportFilter,
   group: CaseReportGroup,
-  field: CaseReportField
+  field: CaseReportField,
+  splitByType: Boolean = false
 )
 
 object CaseReport {
   private val reportGroupKey = "report_group"
   private val reportFieldKey = "report_field"
+  private val splitByTypeKey = "split_by_type"
```

Comment:
We should try and keep reports generic by using multiple report_groups rather than extra fields

Reply:
I'll have a look at doing that

Reply:
I did this on a branch last week, I'll push it up to branch `groupbymulti`

Reply:
Pulled your stuff in on backend, will make it match up on front end